app/i18n/locales/en-US.json

```diff
@@ -479,5 +481,21 @@
   "wallet.transaction.type.intrawallet": "{currency} intrawallet transaction",
   "wallet.transaction.type.multiparty": "{currency} multiparty transaction",
   "widgets.explorer.tooltip": "Go to {websiteName}",
-  "widgets.warningBox.headerText": "ATTENTION:"
-}
\ No newline at end of file
+  "widgets.warningBox.headerText": "ATTENTION:",
+  "yoroiTransfer.error.noTransferTxError": "There is no transfer transaction to send.",
+  "yoroiTransfer.errorPage.title.label": "Unable to transfer from another wallet",
+  "yoroiTransfer.error.transferFundsError": "Unable to transfer funds.",
+  "yoroiTransfer.form.instructions.step0.text": "Enter the 15-word recovery phrase used to back up anothe wallet to restore the balance and transfer all the funds to current wallet.",
+  "yoroiTransfer.start.instructions.button.label": "next",
+  "yoroiTransfer.start.instructions.text": "Transfer funds from another wallet.",
+  "yoroiTransfer.start.instructions.title": "instruction",
+  "yoroiTransfer.successPage.text": "Your funds were successfully transfered.",
+  "yoroiTransfer.successPage.title": "success",
+  "yoroiTransfer.summary.addressFrom.subLabel": "Wallet Addresses",
+  "yoroiTransfer.waiting.checkingAddresses.generatingTx": "Generating transfer transaction",
+  "yoroiTransfer.waiting.progressInfo.checkingAddresses": "Checking addresses funds",
+  "yoroiTransfer.waiting.progressInfo.restoringAddresses": "Fetching addresses",
+  "yoroiTransfer.waiting.title.label": "Wallet is being restored",
+  "yoroiTransfer.waiting.warning": "⚠️ This may take a long time or fail on poor internet connections"
```

Comment:
You mean #739, right?

Reply:
Just realized this text is actually not used. Daedalus transfer has this warning because it needs to fetch the whole UTXO set (right?), which isn't the case for transferring from another wallet

Reply:
Yeah it's not as big of a deal in this situation since the amount you have to transfer is multiple orders of magnitude smaller